libtextsecure/outgoing_message.js

```diff
@@ -368,15 +368,16 @@ OutgoingMessage.prototype = {
     return this.getStaleDeviceIdsForNumber(number).then(updateDevices =>
       this.getKeysForNumber(number, updateDevices)
         .then(async (keysFound) =>  {
+          const conversation = ConversationController.get(number);
           let attachPrekeys = false;
           if (!keysFound)
           {
             log.info("Fallback encryption enabled");
+            conversation.onFriendRequestSent();
```

Comment:
This should probably be moved down to after "reloadDevicesAndSend" returns a confirmation to prevent a conversation being locked before a friend request has actually been sent

Reply:
Good point!